schemas/content/product.schema.json

```diff
@@ -60,81 +53,68 @@
           "description":
             "The internal unique ID of the product in the commerce backend system."
         },
-        "xdm:masterProductSKU": {
-          "title": "Master Product SKU",
-          "type": "string",
+        "xdm:prices": {
```

Comment:
Best to make this a abstract schema so it can be easily reused and extended.

Reply:
Ok, I will convert this into an abstract schema. Which leads to the follow up questions:

* should `product-attribute`schema be abstract as well? I think so yes.
* should the abstract schemas have `product-` prefix like product-attribute, product-price or no prefix?
* what is the best location of these sub schemas used by product? Still be `content` or better `data`?